cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -1414,6 +1444,60 @@ define(['jquery', 'edx-ui-toolkit/js/utils/spec-helpers/ajax-helpers', 'common/j
                     );
                 });
 
+                describe('Show correctness setting set as expected.', function() {
+                    var setShowCorrectness = function(show_correctness) {
+                        $('input[name=show-correctness][value=' + show_correctness + ']').click();
+                    }, expectWarningShown = function(shown) {
+                        expect($('.show-correctness .tip-warning').is(':visible')).toBe(shown);
+                    };
+
+                    describe('Show correctness set by subsection metadata.', function() {
+                        $.each(['always', 'never', 'past_due'], function(index, show_correctness) {
```

Comment:
Hear that..  Though it looks like the simple `for` loop you recommended initially is [more performant](https://coderwall.com/p/kvzbpa/don-t-use-array-foreach-use-for-instead) than either `jQuery.each` or `forEach`.